public/js/monitor.js

```diff
@@ -57,34 +57,63 @@ function isValidEmail(val) {
   return re.test(String(val).toLowerCase());
 }
 
+function addClass(selector, className){
+  if(document.querySelector(selector)){
+    document.querySelector(selector).classList.add(className);
+  }
```

Comment:
I think you can probably cache the results and save a DOM lookup here...

```js
  const el = document.querySelector(selector);
  if (el) {
    el.classList.add(className);
  }
```

... and same w/ the `removeClass()` function below too.

Reply:
👍